src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ContentReviewModule.java

```diff
@@ -35,9 +39,32 @@ public boolean isSubmitLinkVisible() {
         try {
             wait.forElementVisible(submitForReviewLink, 3, 1);
             isLinkVisible = true;
-        } catch(TimeoutException e) {
+        } catch(TimeoutException exception) {
             isLinkVisible = false;
         }
         return isModuleVisible() && isLinkVisible;
     }
+
+    public boolean isEnableTestModeButtonVisible() {
+        boolean isButtonVisible = false;
+        try {
+            wait.forElementVisible(enableTestModeButton, 5, 1);
```

Comment:
why instead of using the default forElementVisible method have you choosen to use the customized timeouts  ? is that necessary ?